Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: deps install all
+
+ifndef NASHPATH
+	$(error $$NASHPATH is not set)
+endif
+
+ifndef GOPATH
+	$(error $$GOPATH is not set)
+endif
+
+all: deps install
+
+deps:
+	go get github.com/c0defellas/enzo/cmd/...
+
+install:
```

Comment:
would it not be a good idea to remove a previous install before ? some older files may remain on the dir as the project evolves. new code that wrongly imports old stuff would even work after make install since the old file would still be present on the system.